components/account/AccountFollowButton.vue

```diff
@@ -65,6 +68,52 @@ const buttonStyle = $computed(() => {
   // If not following, use a button style
   return 'text-inverted bg-primary border-primary'
 })
+
+const dataGlean = $computed(() => {
+  if (!gleanContext)
+    return undefined
+
+  let action
+  if (relationship?.blocking)
+    action = 'follow.unblock'
+  else if (relationship?.muting)
+    action = 'follow.unmute'
+  else if (relationship ? relationship.following : context === 'following')
+    action = 'follow.unfollow'
+  else if (relationship?.requested)
+    action = 'follow.withdraw-follow-request'
+  else if ((relationship ? relationship.followedBy : context === 'followedBy') && account.locked)
```

Comment:
this horrendous conditional comes from somewhere else in the `<template>` code below